src/Entity/Team/Member.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App\Entity\Team;
+
+use App\Entity\Adherent;
+use App\Entity\EntityIdentityTrait;
+use App\Entity\EntityTimestampableTrait;
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\Team\MemberRepository")
+ * @ORM\Table(uniqueConstraints={
```

Comment:
What about naming the table as `team_member`?
```suggestion
 * @ORM\Table(name="team_member", uniqueConstraints={
```